Week-2/animation/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<head>
+  <link href="style.css" rel="stylesheet" type="text/css" media="all" />
+  <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+  <div id="grandparent">
+    <div id="parent">
+      <div class="child-1"></div>
+      <div class="child-2"></div>
+      <div class="child-3"></div>
+      <div class="child-4"></div>
```

Comment:
You can give class name child to all so that when you put styles it applies to all. If some child requires different styles then give another class name as child-3, 4 etc.